src/Nethermind/Nethermind.Db.Rocks/Config/PerTableDbConfig.cs

```diff
@@ -38,7 +38,7 @@ private void EnsureConfigIsAvailable(string propertyName)
         foreach (var prefix in _prefixes)
         {
             string prefixed = string.Concat(prefix, propertyName);
-            if (type.GetProperty(prefixed, BindingFlags.Public | BindingFlags.Instance) is null)
+            if (GetProperty(type, prefixed) is null)
```

Comment:
Changed code will now add `BindingFlags.IgnoreCase` here.
Before, config validation wasn't using this flag. while actual config reading was.